front/src/layouts/layout-footer/layout-footer.layout.js

```diff
@@ -0,0 +1,8 @@
+import { Logo } from '@components';
+
+export default {
+  name: 'ff-footer',
+  components: {
+    'ff-logo': Logo
```

Comment:
the same as with footer, change in logo component name into ff-logo and you don't need specify this name here in declarations of outer components